README.md

```diff
@@ -389,10 +389,10 @@ cargo-dist is self-hosting, so just follow the usual [usage instructions](#usage
 The CI is (re)generated with:
 
 ```
-cargo dist generate-ci github --installer=github-shell --installer=github-powershell
+cargo dist generate-ci
 ```
 
-**Including the installers is very important, as all the CI scripts cargo-dist generates for *other* projects will bootstrap dist with those installers.**
+**NOTE: if you want to update the version of cargo-dist used in the CI, you need to update cargo-dist-version in the root Cargo.toml! I keep forgetting this and should add a feature to help this this! Yeah! Looking at you, Future Me Who Still Hasn't Done This!**
```

Comment:
this note feels like it might be worth filing an issue for tackling in the next release and/or documenting more explicitly

Reply:
https://github.com/axodotdev/cargo-dist/issues/141